HCCGo/app/js/ConnectionService.js

```diff
@@ -872,14 +872,24 @@ connectionModule.factory('connectionService',['$log', '$q', '$routeParams', '$lo
                 }
        });
 
-   }
+   };
 
+   var quickDownload = function(remotePath, localPath) {
+       connectionList[getClusterContext()].sftp(function (err, sftp) {
+           if (err) {
+               return;
```

Comment:
You should probably capture this error.  Probably using a promise, and reject the promise here.